cmd/tempo/main.go

```diff
@@ -66,6 +66,17 @@ func main() {
 	// Allocate a block of memory to alter GC behaviour. See https://github.com/golang/go/issues/23044
 	ballast := make([]byte, *ballastMBs*1024*1024)
 
+	// Warn the user for suspect configurations
```

Comment:
Let's put this in its own method and call it from main().  It will help with readability.

Reply:
I put it on `main()` because it's a one-time thing. But I agree, I'll move it to its own method.